### ISSUE TITLE
Page editor autocomplete for inputs - Implement the logic for suggestions

### ISSUE THREAD

**Issue by BLoe** [pixiebrix/pixiebrix-extension#2071]
Considerations:
- Do we autocomplete from the trace context of the last run?
- Before the first run
  - Can we run pure blocks and do speculative logic around variable names?
- When the configs are out of date, continue to use last trace run no matter what, for now, until we see user issues
Nice to have:
- Searching based on actual data content/value in addition to variable names 

Blocked by: #2040

**Comment by twschiller** [comment]
Shipped as variable popover
